sssom/prefix_reconciliation.yaml

```diff
@@ -0,0 +1,5 @@
+custom_prefix_map:
```

Comment:
we do have `reconciliation` as a parameter to `merge` but that does something else. See [here](https://github.com/mapping-commons/sssom-py/blob/bf9c32b7fa346361d7e4b5993b00204fa6060d7f/sssom/util.py#L508). That ha nothing to do with `prefix_reconciliation`. So you're saying the `prefix_reconcilation` logic should not reside in `sssom-py` at all ? I thought we had decided to keep it here. I'm confused.

Reply:
no the prefix reconciliation logic should definitely reside in soom, but not the prefixes to be reconciled (the yaml file) this should be passed in as a parameter like `--reconcile-prefixes`, not baked into sssom-py. This will change from case to case!

Reply:
got it!

Reply:
I've implemented this only for the `merge` CLI as of now. Is that what' expected?